#include<iostream>

using namespace std;


/*
int main()
{
	int a = 100, b;

	cout << "Input number: ";
	cin >> b;

	try
	{
		//예외를 검사하고 던진다
		if (b == 0)
		{
			throw b;
		}
		else
		{
			cout << a / b << endl;
		}

		
	}
	//try문 블록 내에서 실행한 각종 연산들의 예외 상황 코드가 집결됨
	//만일 예외 상황이 여러 가지라면 catch문 블록 내에서 다시 switch-case문이나 다중 if문을 이용해 상황을 분류하고 각각에 맞게 코드 작성
	catch (int nExp)
	{
		//예외를 받아 처리한다
		cout << "ERROR: " << nExp << "으로 나눌 수 없습니다" << endl;
	}

	//구조화 된 예외처리는 어디까지나 선택 사항
	//본인의 상황에 맞는 것을 선택하면 됨
}*/